src/client/styles/components/RequestDigitization.scss

```diff
@@ -18,6 +18,7 @@
     box-shadow: -1px 0 1px 0 rgba(0,0,0,0.3);
     padding: 0.35rem 0.5rem;
     text-align: left;
+    z-index: 100;
```

Comment:
Is this just for safety or is there a known case that can cause something else to appear in front of the form?

Reply:
Yeah, Chris found it:  
https://jira.nypl.org/browse/SFR-89

It comes from the desire to do this in as quick-and-dirty of a way as possible ... a more proper way to do this would be to place it in the center of the screen with all the correct screen positioning calculations.  

The hope is that when this gets implemented properly (not as a modal), it would be using a different class and this class will be eliminated, because z-index like this is ... generally a hack.

Reply:
haha yeah, z-index stuff can definitely get out of hand, we aren't using too many modals though and this definitely works to address the issue for now